pep-0458.txt

```diff
@@ -408,14 +408,20 @@ have any of the keys required to sign for projects.  However, it does not
 protect projects from attackers who have compromised PyPI, since they can
 then manipulate TUF metadata using the keys stored online.
 
-This PEP proposes that the *bin-n* roles sign for all
-PyPI projects with online keys.  The *targets* role, which only signs with an
-offline key, MUST delegate all PyPI projects to the *bins* role.  This means
-that when a package manager such as pip (i.e., using TUF) downloads a
-distribution from a project on PyPI, it will consult the *bins* role about the
-TUF metadata for the project.  If no *bin-n* roles delegated by *bins* specify the
-project's distribution, then the project is considered to be non-existent on
-PyPI.
+This PEP proposes that the *bin-n* roles sign for all PyPI projects with online
+keys. They MUST all be delegated by the upper-level bins* role, which is signed
```

Comment:
```suggestion
keys. They MUST all be delegated by the upper-level *bins* role, which is signed
```

Reply:
Fixed in a force push. Thanks for the eyes, @joshuagl.

Reply:
In the second sentence, its not clear who or what is "they" so see suggestion
```suggestion
keys. These *bin-n* roles MUST all be delegated by the upper-level bins* role, which is signed
```

Reply:
Otherwise, I think this text is OK.